pkg/monitor/cluster/clusterversions.go

```diff
@@ -65,3 +72,20 @@ func desiredVersion(cv *configv1.ClusterVersion) string {
 
 	return cv.Status.Desired.Version
 }
+
+// baselineVersion checks the upgradeStreams for possible upgrade of the cluster
+// when the upgrade is possible withing the current Y version, return true and closest upgrade stream version
+func baselineVersion(cv *configv1.ClusterVersion, streams []*version.Stream) (string, bool) {
```

Comment:
can simplify this nicely by removing the bool return value - just return "" or the version.  Then the if check and `dims :=` change above is no longer needed.